python/src/ngraph/ops.py

```diff
@@ -673,15 +702,26 @@ def sqrt(node, name=None):  # type: (NodeInput, str) -> Node
     return Sqrt(node)
 
 
+@unary_op
+def erf(node, name=None):  # type: (NodeInput, str) -> Node
+    """Return node which calculates Gauss error function element-wise with given tensor.
+
+    :param node: The node providing data for operation.
+    :param name: The optional name for new output node.
+    :return: The new node performing element-wise Erf operation.
+    """
+    return _get_node_factory().create('Erf', [as_node(node)])
```

Comment:
`as_node` is redundant here. Please take a look into `decorators.py` at `def unary_op(node_factory_function):`.

Reply:
Done